### PR TITLE
Verify present button Log Out after loging

### PR DESCRIPTION
**Pull request by TyshchenkoRoma**

### CODE REVIEW

**Review by MelnykNikita — APPROVED** [review]

File: src/test/java/rv/atqc/gttesting/logOutSystem/LogOutChrome.java [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+import org.testng.Assert;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+public class LogOutChrome {
+    private WebDriver driver;
+
+    @Test
+    public void identifyLogOutButtonChrome() {
```
Comment (dmatsev) [inline src/test/java/rv/atqc/gttesting/logOutSystem/LogOutChrome.java]: I would prefer the title "CheckLogOutButtonTitleInChrome()"

File: src/test/java/rv/atqc/gttesting/logOutSystem/LogOutChrome.java [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+        clickOnLeftTopGreenButton();
+        insertLogin("17141@rambler.ru");
+        insertPassword("12345678");
+        clickOnLoginButton();
+        clickOnLeftTopGreenButton();
+        Assert.assertEquals(getLogOutText(), "Log out");
+    }
+
+    @Test
+    public void identifyTitleForgotPasswordAfterLogingOutChrome() {
```
Comment (dmatsev) [inline src/test/java/rv/atqc/gttesting/logOutSystem/LogOutChrome.java]: I would prefer the title "CheckForgotPasswordLinkTitleInChrome()"

File: src/test/java/rv/atqc/gttesting/logOutSystem/LogOutChrome.java [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+public class LogOutChrome {
+    private WebDriver driver;
+
+    @Test
+    public void identifyLogOutButtonChrome() {
+        clickOnLeftTopGreenButton();
+        insertLogin("17141@rambler.ru");
```
Comment (dmatsev) [inline src/test/java/rv/atqc/gttesting/logOutSystem/LogOutChrome.java]: move it to the string const and reuse in other cases

File: src/test/java/rv/atqc/gttesting/logOutSystem/LogOutChrome.java [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+        insertPassword("12345678");
```
Comment (dmatsev) [inline src/test/java/rv/atqc/gttesting/logOutSystem/LogOutChrome.java]: move it to the string const and reuse in other cases

File: src/test/java/rv/atqc/gttesting/logOutSystem/LogOutChrome.java [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+        clickOnLoginButton();
+        clickOnLeftTopGreenButton();
+        Assert.assertEquals(getLogOutText(), "Log out");
+    }
+
+    @Test
+    public void identifyTitleForgotPasswordAfterLogingOutChrome() {
+        clickOnLeftTopGreenButton();
+        insertLogin("17141@rambler.ru");
```
Comment (dmatsev) [inline src/test/java/rv/atqc/gttesting/logOutSystem/LogOutChrome.java]: move it to the string const and reuse in other cases

File: src/test/java/rv/atqc/gttesting/logOutSystem/LogOutChrome.java [hunk 6]
```diff
@@ -0,0 +1,76 @@
...
+        insertPassword("12345678");
```
Comment (dmatsev) [inline src/test/java/rv/atqc/gttesting/logOutSystem/LogOutChrome.java]: move it to the string const and reuse in other cases

**Review by dmatsev — COMMENTED** [review]
please make changes and notify me when it will be done

**Review by dmatsev — APPROVED** [review]

**Review by romanukhaav — APPROVED** [review]
Good work!
I have no to add or change anything!

**Pull request merged**